NodeFirefox/Dockerfile.txt

```diff
@@ -4,10 +4,11 @@ USER root
 # Firefox
 #=========
 ARG FIREFOX_VERSION=57.0.2
-RUN apt-get update -qqy \
+RUN FIREFOX_DOWNLOAD_URL=$(if [ $FIREFOX_VERSION = "nightly" ]; then echo "https://download.mozilla.org/?product=firefox-nightly-latest-ssl&os=linux64&lang=en-US"; else echo "https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2"; fi) \
+  && apt-get update -qqy \
   && apt-get -qqy --no-install-recommends install firefox \
```

Comment:
I noticed this in there. Is this needed if we get Firefox from Mozilla directly ?